include/ledstripeffect.h

```diff
@@ -57,9 +61,9 @@ class LEDStripEffect : public IJSONSerializable
     int    _effectNumber;
     bool   _enabled = true;
     size_t _maximumEffectTime = SIZE_MAX;
-    std::vector<std::reference_wrapper<SettingSpec>, psram_allocator<SettingSpec>> _settingSpecs;

```

Comment:
Lost the psram allocator again?

Reply:
Nope. I explained this in one of my own comments. Again, in short:

- I don't think allocating the reference_wrapper in PSRAM will make a dent in plain RAM memory usage - as I don't think it will for unique_ptr or shared_ptr. That's true as long as the objects they point/refer to are allocated on PSRAM, which the SettingSpec instances are.
- The code as it stood was invalid. The T of the vector (std::reference_wrapper<SettingSpec>) must match the T of the allocator, which it didn't.